TLM/TLM/UI/Textures/SpeedLimitTextures.cs

```diff
@@ -2,13 +2,129 @@ namespace TrafficManager.UI.Textures {
     using static TextureResources;
     using System.Collections.Generic;
     using System;
+    using System.Linq;
+    using CSUtil.Commons;
     using TrafficManager.API.Traffic.Data;
     using TrafficManager.State;
-    using TrafficManager.UI.SubTools.SpeedLimits;
     using TrafficManager.Util;
     using UnityEngine;
 
     public static class SpeedLimitTextures {
+        public class RoadSignTheme {
+            private IntVector2 TextureSize;
+
+            /// <summary>Speed limit signs from 5 to 140 km (from 5 to 90 mph) and zero for no limit.</summary>
+            public readonly Dictionary<int, Texture2D> Textures = new();
+
+            /// <summary>This list of required speed signs is used for loading.</summary>
+            private List<int> SignValues = new();
+
+            private string PathPrefix;
+
+            public bool SupportsKmph;
```

Comment:
why are these public fields that can be changed from the outside?

Reply:
Changed to readonly, they are accessed from General options tab.